pycbc/distributions/constraints.py

```diff
@@ -60,7 +79,27 @@ def _constraint(self, params):
         return params[self.constraint_arg]
 
 
+class ConvexHull(Constraint):
+    """Pre defined constraint for core-collapse waveforms that checks
+    whether a given set of coefficients lie within the convex hull of
+    the coefficients of the principal component basis vectors.
+    """
+    name = "supernovae_convex_hull"
+    required_parameters = ["coeff_0", "coeff_1"]
```

Comment:
Add an `__init__` function to execute the code you have above. You can use `super` to call the parent class `__init__`; e.g.:
```
def __init__(self, constraint_arg, transforms=None, **kwargs):
    super(SupernovaeConvexHull, self).__init__(constraint_arg, transforms=transforms, **kwargs) # <- this will execute Constraint.__init__, thereby setting all of the attributes that are set there
   [ your block of code from above ]
```